iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/match/AffOrgMatcherFactory.java

```diff
@@ -97,12 +98,17 @@ public static AffOrgMatcher createDocOrgRelationMatcher(JavaSparkContext sparkCo
      * Creates {@link AffOrgMatchVoter}s for {@link #createDocOrgRelationMatcher(JavaSparkContext, String, String, String, Float)} 
      */
     public static ImmutableList<AffOrgMatchVoter> createDocOrgRelationMatcherVoters() {
+
+        FittingOrgWordsMatchVoter fitOrgWordsMatchVoter = new FittingOrgWordsMatchVoter(ImmutableList.of(','), 3, 0.8f, 0.9f);
```

Comment:
As to the spring, yes, I thought about it but I'm not sure if it is a good idea, and certainly, not here along with this task.
And as far as parameters are concerned, no, I don't think we can do something in real time. Do you have any ideas?